float64 x
float64 y
float64 z
float64 side_x
float64 side_y
float64 interval
---
bool ok
